.circleci/config.yml

```diff
@@ -136,10 +136,11 @@ jobs:
             gotestsum --junitfile "$TEST_RESULTS/gotestsum-report.xml" -- ./... -p 1 -timeout 20m -failfast \
               -enable-enterprise \
               -enable-multi-cluster \
+              -run TestControllerNamespaces\
```

Comment:
@ashwin-venkatesh I was thinking of only running the controller tests on our base branch so we can iterate more quickly